tomcat-mixin/dashboards/tomcat-hosts.libsonnet

```diff
@@ -0,0 +1,489 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+
+local dashboardUid = 'hosts';
+
+local promDatasourceName = 'prometheus_datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local sessionsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum(increase(tomcat_session_sessioncounter_total{job=~"$job", instance=~"$instance", host=~"$host", context=~"$context"}[$__interval:])) by (job, instance)',
+      datasource=promDatasource,
+      legendFormat='{{job}} - {{instance}} - total sessions',
```

Comment:
i know this is obnoxious feedback and i'm sorry (because this is contrary to what i originally suggested), but based on my last round of feedback from grafana, i would actually remove the job from the legend. sounds like they would prefer less clutter and just use the selectors if there is any confusion